mina-archive/values.yaml

```diff
@@ -19,6 +19,7 @@ archive:
     enabled: false
     s3Bucket: 673156464838-mina-archive-node-backups
   bootstrapDB: true
+  bootstrapDBAnnotations: {}
```

Comment:
This is fine in the current context as it is not versioned, and so we can't afford breaking changes
in the future, but ideally, it would be good to refactor this way:
```
bootstrapDB:
    enable: true
    podAnnotations: {}
```

Reply:
I agree.

I'm trying to resolve an apparent bug with ArgoCD and helmfile. For some reason it stops rendering the Deployment template when adding .metadata.annotaions field. For the Job it does not seem to care.


Still digging into the issue.